moped-editor/src/views/dev/LookupsView/index.js

```diff
@@ -142,6 +148,19 @@ const LookupsView = () => {
                       <LinkIcon fontSize="small" />
                     </IconButton>
                   </Grid>
+                  <Grid item>
+                    <IconButton
+                      component={Link}
+                      to={"#"}
+                      onClick={(e) => {
+                        e.preventDefault();
+                        scrollToTable("_scroll_to_top", refs);
+                        history.replace("");
+                      }}
+                    >
+                      <ArrowUpwardIcon fontSize="small" />
```

Comment:
would it feel less awkward to you if it had a tooltip that said "Scroll to top" ?